### DISCUSSION TITLE
执行demo抛出错

### DISCUSSION THREAD

**Discussion by zhugexinxin** [RapidAI/TableStructureRec#27] [Q&A]
<img width="794" alt="image" src="https://github.com/user-attachments/assets/d8c9f80c-b967-4c3e-b095-f18dce3e6d40">
<img width="1020" alt="image" src="https://github.com/user-attachments/assets/0a6f8d0a-74f8-4933-b045-7fdb9f09bd33">
执行demo抛出这个错

**Comment by Joker1212** [comment]
已经修复了哈，再试试呢
